### PR TITLE
chore: make navbar more responsive

### PR DESCRIPTION
**Pull request by fionnachan**

### CODE REVIEW

File: website/src/css/custom.css [hunk 1]
```diff
@@ -789,3 +815,7 @@ article a:not([data-quicklook-from]):not(.footnote-ref) {
   display: flex;
   justify-content: center;
 }
+.navbar__item {
+  font-size: clamp(0.8rem, 1.25vw, 1rem);
+  line-height: 1.5;
+}
```
Comment (fionnachan) [inline website/src/css/custom.css:821]: actual change

File: website/src/css/custom.css [hunk 2]
```diff
@@ -139,8 +154,9 @@
   --tab-active-color: #3578e5;
   --tab-active-color-hover: hsl(217, 58%, 36%);
   --tab-gray: #ddd;
+  --ifm-navbar-item-padding-horizontal: 0.6rem;
```
Comment (fionnachan) [inline website/src/css/custom.css:157]: actual change

**Review by fionnachan — COMMENTED** [review]
only 2 changes
others are from prettier

**Comment by symbolpunk** [comment]
Should these changes be applied to the `custom.less` file that sits next to the CSS file?

The former generates the latter; unless I'm mistaken, these changes will be overwritten whenever the LESS file is compiled into CSS (locally - it's not hooked up to ci/cd afaik).

**Comment by chrisco512** [comment]
LGTM

**Review by chrisco512 — APPROVED** [review]
LGTM

**Pull request merged**